### PR TITLE
Added link to README badge which forwards on click to the ci

### PR DESCRIPTION
**Pull request by EMaksy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Formula CI](https://github.com/SUSE/drbd-formula/drbd-formula/workflows/Formula%20CI/badge.svg)
+[![Formula CI](https://github.com/SUSE/drbd-formula/actions/workflows/drbd-formula-ci.yml/badge.svg)](https://github.com/SUSEyy/drbd-formula/actions/workflows/drbd-formula-ci.yml)
```
Comment (nick-wang) [inline README.md]: @SchleichsSalaticus Thanks for the commit.
Maybe there is a typo in the URL which direct to an not existed page. (I guess s/SUSEyy/SUSE/)

**Review by nick-wang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EMaksy) [reply README.md]: Yes, thank you :+1:   I corrected the typo.  The funny thing is it was displaying the badge correct on my fork.

**Review by arbulu89 — APPROVED** [review]
Looks good @SchleichsSalaticus . Thanks

**Review by nick-wang — APPROVED** [review]

**Comment by nick-wang** [comment]
@SchleichsSalaticus Looks good now, Thanks!

**Pull request merged**